### ISSUE TITLE
Jetpack Connection: close button in connect account page has no accessible name

### ISSUE THREAD

**Issue by monsieur-z** [Automattic/jetpack#35483]
In the "Connect Account" page, the close button has no accessible name. Screen readers have no label to read and fall back to reading the URL. From a UI perspective, this type of button is probably not the most appropriate since the content of the page is actually rendered inside the page and not in a modal.

Issue reported in p1HpG7-qVW-p2.

**Comment by monsieur-z** [comment]
Issue is fixed in https://github.com/Automattic/jetpack/pull/35712.

Putting this as blocked for now, as a further improvement could be to not use a modal in this screen. Waiting for Design's feedback: p6TEKc-7DI-p2.

**Comment by monsieur-z** [comment]
Closing the issue. I'll open a new one for improvements if need be.
